src/Compilers/Core/Portable/Symbols/IModuleSymbolInternal.cs

```diff
@@ -2,11 +2,14 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
-
 namespace Microsoft.CodeAnalysis.Symbols
 {
     internal interface IModuleSymbolInternal : ISymbolInternal
     {
+        /// <summary>
+        /// Gets the first unsupported CompilerFeatureRequired string, if one exists. Null if there are none.
+        /// </summary>
+        string? GetUnsupportedCompilerFeature();
+        new IAssemblySymbolInternal ContainingAssembly { get; }
```

Comment:
>new IAssemblySymbolInternal ContainingAssembly { get; } [](http://example.com/codeflow?start=8&length=55)

I am not sure why we need to add this API since ```ISymbolInternal``` already has it.